### PR TITLE
MkII PCB

### PR DESCRIPTION
**Pull request by ADBeta**
Moved MkI PCB to `PCB/MkI` folder
Added MkII PCB Files
Changed README format and added accreditation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
- - `1-9` to quickly set the voltage/current target in multiples of 1000mV/100mA
- - ~~Konami code makes the unit self distruct.~~ Feature removed due to misuse.
+> [!NOTE]
+> Need to improve the command list and handler
+
+- `0` to turn the supply off
+- `1-9` to set the target Voltage/Current in multiples of 1000mV/100mA
+- `c` to switch to Constant Current Mode 
+- `v` to switch to Constant Voltage Mode
+- `+` to increase the target Voltage/Current depending on mode, in 50mV/25mA increments
```
Comment (BogdanTheGeek) [inline README.md]: forgot `-` to decrease.
Should we even mention this? its only for debugging without usb.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
 
 # Build
 ```sh
 export CH32V003FUN="path/to/ch32v003fun"
 make
 ```
+
+----
+(c) 2024  
+Bogdan Ionescu
```
Comment (BogdanTheGeek) [inline README.md]: use github links and add under a contributors list.

**Review by BogdanTheGeek — COMMENTED** [review]
I think you also need to rebase or drop the changes to the firmware, bootloader and webui as they have already been merged

**Comment by BogdanTheGeek** [comment]
fixes #3

**Comment by ADBeta** [comment]
Updated to match current project head

**Pull request closed without merging**